example/app.js

```diff
@@ -67,6 +69,16 @@ class App extends React.Component {
     })
   }
 
+  getInitialFormValues(fields) {
+    let values = {};
+
+    Object.keys(fields).forEach(name => {
+      values[name] = fields[name].value;
```

Comment:
this should probably be:
```
let value = fields[name].value;
if (typeof value !== 'undefined') {
  values[name] = value;
}
```
so that values like `false` or `0` are properly stored, but a field without this property explicit set to something doesn't do anything to the `values` object.